src/output/mod.rs

```diff
@@ -1,3 +1,4 @@
+#[allow(unused_imports)]
```

Comment:
Just a nitpick, but since it's just netbsd complaining we could be more specific here I think:
```rust
#[cfg_attr(target_os = "netbsd", allow(unused_imports))]
```

Reply:
I agree with you, should be done in the latest commit.